frontend/src/components/skeletons/featured-post-card-skeleton.tsx

```diff
@@ -2,32 +2,32 @@ import { Skeleton } from '@/components/ui/skeleton';
 
 export const FeaturedPostCardSkeleton = () => {
   return (
-    <div className="flex rounded-lg bg-light dark:bg-dark-card">
+    <div className="flex flex-col md:flex-row rounded-lg bg-light dark:bg-dark-card">
       {/* for image */}
-      <div className="w-1/3">
-        <Skeleton className="h-full w-full rounded-lg bg-slate-200 dark:bg-slate-700" />
+      <div className="w-full md:w-1/3">
+        <Skeleton className="h-48 md:h-full w-full rounded-lg bg-slate-200 dark:bg-slate-700" />
       </div>
-      <div className="flex h-48 w-2/3 gap-4 rounded-lg p-3">
+      <div className="flex flex-col md:w-2/3 gap-4 rounded-lg p-3">
         <div className="flex w-full flex-col gap-3">
           {/* for title */}
-          <Skeleton className="h-6 w-11/12 bg-slate-200 dark:bg-slate-700" />
-          <Skeleton className="h-6 w-2/3 bg-slate-200 dark:bg-slate-700" />
+          <Skeleton className="h-6 w-full md:w-11/12 bg-slate-200 dark:bg-slate-700" />
+          <Skeleton className="h-6 w-full md:w-2/3 bg-slate-200 dark:bg-slate-700" />
           <div className="flex flex-wrap gap-2">
```

Comment:
Same comment as above, in the actual post card we are aiming sm: and lg: , but in skeleton we should also do the same for consistency, choosing md: will fuel out of sync design.